themes/codefor-theme/layouts/labs/single.html

```diff
@@ -13,6 +13,13 @@ <h1 class="headline-brackets blue flex-grow-1 m-0">{{.Title}} </h1>
     {{ partial "hero-block-center.html" (dict "context" . "txt" .Params.Hero ) }}
     {{ end }}
 
+    {{ if .Params.inactive }}
+        <div class="alert alert-warning mt-4" role="alert">
+            Dieses Lab ist nicht mehr aktiv. <br />  Wenn du Fragen zum Netzwerk hast, dann schreib
+            uns gerne direkt an info@codefor.de.
+        </div>
+    {{ end }}
```

Comment:
Please change text to: 

Dieses Lab ist gerade nicht aktiv. Das bedeutet, dass aktuell keine Treffen stattfinden und die Kontaktpersonen ihr Ehrenamt derzeit ruhen lassen. Wenn du Fragen zum Netzwerk hast, dann schreib uns gern an info@codefor.de.